modules/i18n/src/translations/en-US/portals/claims.ts

```diff
@@ -451,6 +451,26 @@ export const claims: ClaimsNS = {
                 placeholder: "Enter an attribute name",
                 requiredErrorMessage: "Attribute name is required"
             },
+            canonicalValues: {
+                hint: "Provide the allowed values for the attribute.",
+                keyLabel: "Display Value",
+                keyRequiredErrorMessage: "Display value is required",
+                valueLabel: "Actual Value",
+                valueRequiredErrorMessage: "Actual value is required"
```

Comment:
Actual value sounds kinda odd, imo.

```suggestion
                valueLabel: "Value",
                valueRequiredErrorMessage: "Value is required"
```